Submissions/AvdheshKumarAKG/Assignment1/BC-Assignment1.py

```diff
@@ -0,0 +1,58 @@
+from hashlib import sha256
+import time
+import hashlib
+class block():
+	def __init__(self, timestamp, data, previousHash = ''):
+
+		self.timestamp = timestamp
+		self.data=data
+		self.previousHash = previousHash
+		self.hash = self.calculateHash()
+
+	def calculateHash(self):
+
+		HASHobj = hashlib.sha256()
+		HASHobj.update(self.data)
```

Comment:
2 comments here,

* You have only hashed the data and not the timestamp and the previousHash
* you have to encode data before hashing, it is necessary